src/features/increase-fee-drawer/components/actions.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+import { Button, Stack } from '@stacks/ui';
+
+import { stxToMicroStx } from '@common/stacks-utils';
+import { LoadingKeys, useLoading } from '@common/hooks/use-loading';
+import { useRawDeserializedTxState, useRawTxIdState } from '@store/transactions/raw.hooks';
+
+interface ActionsProps {
+  formik: FormikProps<{ fee: number }>;
+}
```

Comment:
Can you use the formik hook here so we don't need to pass the prop/repeat the type? Also `Actions` isn't such a descriptive name.

Reply:
Yep, will change. I don't have much experience with Formik so will look at using the hooks.